### ISSUE TITLE
When we come across a list containing more than one item, the total number of items is not displayed in Braille.

### ISSUE THREAD

**Issue by ericdq** [AAClause/BrailleExtender#161]
In the latest NVDA updates, when we come across a list containing more than one item, the total number of items appears after the list indicator, just as it would when read aloud. For me, it appears as lst10, for example.

This information isn’t carried over to Braille Extender, perhaps because this marker hasn’t been included amongst the elements that can be represented.

It would be useful to incorporate it, as it is a feature that has been missing from NVDA for a long time. If I remember correctly, I think this feature was available in Braille Extender at some point.

Thank you!

**Comment by josephsl** [comment]
Hi,

Technical: yes, this feature was available from Braille Extender for a long time (lst(itemCount)). This is unavailable in Braille Extender because the add-on overrides parts of the braille support module responsible for this representation.

Thanks.
